dp-hello-world-controller/ci/scripts/audit.sh

```diff
@@ -1,15 +1,7 @@
----
-platform: linux
+#!/bin/bash -eux
 
-image_resource:
-  type: docker-image
-  source:
-    repository: onsdigital/dp-concourse-tools-nancy
-    tag: latest
+export cwd=$(pwd)
 
-inputs:
-  - name: dp-hello-world-controller
-    path: dp-hello-world-controller
-
-run:
-  path: dp-hello-world-controller/ci/scripts/audit.sh 
\ No newline at end of file
+pushd $cwd/dp-hello-world-controller
```

Comment:
Based on previous comment, once removed exported variable cwd this line needs updating to:

```suggestion
pushd dp-hello-world-controller
```